index.html

```diff
@@ -15,7 +15,6 @@
     <p class="lead">Druid is a fast column-oriented distributed data store</p>
     <p><a class="btn btn-lg btn-outline-inverse" href="downloads.html"><span class="fa fa-download"></span> Download</a>
     <a class="btn btn-lg btn-outline-inverse" href="https://github.com/druid-io/druid/"><span class="fa fa-github"></span> GitHub</a></p>
-    <p class="small text-muted">latest stable release {{ site.druid_stable_version }}</p>
```

Comment:
maybe we can add it to the download button? If we only show the last 4-5 news items, it's possible that the latest release will be hidden